testing/internal/e2e/boundary/account.go

```diff
@@ -0,0 +1,75 @@
+package boundary
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/accounts"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/go-secure-stdlib/base62"
+	"github.com/stretchr/testify/require"
+)
+
+// CreateNewAccountApi creates a new account using the go api.
+// Returns the id of the new account as well as the password that was generated
+func CreateNewAccountApi(t testing.TB, ctx context.Context, client *api.Client, loginName string) (string, string) {
+	c, err := loadConfig()
+	require.NoError(t, err)
+
+	aClient := accounts.NewClient(client)
+	password, err := base62.Random(16)
+	require.NoError(t, err)
+	newAccountResult, err := aClient.Create(ctx, c.AuthMethodId,
+		accounts.WithPasswordAccountLoginName(loginName),
+		accounts.WithPasswordAccountPassword(password),
+	)
+	require.NoError(t, err)
+
+	newAccountId := newAccountResult.Item.Id
+	t.Logf("Create Account: %s", newAccountId)
+	t.Cleanup(func() {
+		_, err := aClient.Delete(ctx, newAccountId)
+		require.NoError(t, err)
+	})
+
+	return newAccountId, password
+}
+
+// CreateNewAccountCli creates a new account using the cli.
+// Returns the id of the new account as well as the password that was generated
+func CreateNewAccountCli(t testing.TB, loginName string) (string, string) {
+	c, err := loadConfig()
+	require.NoError(t, err)
+
+	ctx := context.Background()
+	password, err := base62.Random(16)
+	require.NoError(t, err)
+	os.Setenv("E2E_TEST_ACCOUNT_PASSWORD", password)
```

Comment:
I don't think this is necessary, but you could consider reintroducing the RunCommand options and let a user set environment variables for the command instead of setting it for the whole process like this.

Reply:
Interesting suggestion. It's probably cleaner to keep the environment variables within just the command process. Implementing it.